libbeat/tests/system/beat/beat.py

```diff
@@ -49,17 +49,33 @@ def __del__(self):
 
 class TestCase(unittest.TestCase):
 
-    def run_mockbeat(self, cmd="../../libbeat.test",
-                     config="mockbeat.yml",
-                     output="mockbeat.log",
-                     extra_args=[],
-                     debug_selectors=[]):
+    @classmethod
+    def setUpClass(self):
+        self.beat_name = "beat"
+        self.build_path = "../../build/system-tests/"
+        self.beat_path = "../../" + self.beat_name + ".test"
+
+    def run_beat(self, cmd=None,
+                 config=None,
+                 output=None,
+                 extra_args=[]):
         """
-        Executes mockbeat
+        Executes beat on an input pcap file.
```

Comment:
Is this comment accurate? The function body doesn't seem to refer to PCAP files, and if it did, it would be very specific to packetbeat, so it would be better to have that in the packetbeat module.